media/css/firefox/family/components/_pro-tip.scss

```diff
@@ -38,6 +58,16 @@
             padding-left: $spacing-lg;
         }
     }
+
+    @media (prefers-reduced-motion: no-preference) {
```

Comment:
**issue (blocking)**: forces layout reflow, creates small but noticeable shift in surrounding grid bg sections
<img width="1346" alt="no-fill-mode" src="https://user-images.githubusercontent.com/19650432/196403289-fe9143b8-c12e-4055-aa82-05102160dda8.png">

**suggestion**: set x-offset transforms as initial position of elements, use animation fill mode to stay at end of animation transform position (this seems to give the browser enough information to prevent reflow)

```
        h2 {
            transform: translate(-50%);
        }

        p {
            transform: translateX(50%)
        }

        h2.animate-pop-in {
            animation-name: slide-left;
            animation-fill-mode: forwards;
        }

        p.animate-pop-in {
            animation-name: slide-right;
            animation-fill-mode: forwards;
        }
```

tested
<img width="917" alt="fill-mode-forwards" src="https://user-images.githubusercontent.com/19650432/196403545-807ee09a-c66d-4550-b579-690c4a09482f.png">

Reply:
Changing to 50% for transform because otherwise the elements start too far offscreen to trigger the intersection observer animation class